mobile_packet_verifier/src/verifier.rs

```diff
@@ -0,0 +1,44 @@
+use chrono::{DateTime, Utc};
+use file_store::mobile_session::DataTransferSessionIngestReport;
+use futures::{Stream, StreamExt};
+use sqlx::{Postgres, Transaction};
+
+#[derive(thiserror::Error, Debug)]
+pub enum VerificationError {
+    #[error("file store error: {0}")]
+    FileStoreError(#[from] file_store::Error),
+    #[error("sqlx error: {0}")]
+    SqlxError(#[from] sqlx::Error),
+    #[error("reports stream dropped")]
+    ReportsStreamDropped,
+}
+
+pub async fn verify(
```

Comment:
is this module going to be doing additional things over time? this doesn't seem to be "verifying" anything

Reply:
It isn't. Not sure what a better module name would be. Accumulate maybe?